### PR TITLE
Chore: IND-77 adapt doc to JT terraform environnement

### PR DESCRIPTION
**Pull request by simonrdz**
# Description

Change module usage to add this repository URL

# References

# Motivation and Context

# How has this been tested

# Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist

- [x] This PR requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] My code is formatted using terraform fmt.
- [x] There is a **README.md** file in my module (I've created/updated if needed) with a sample usage for this module
- [x] I have referenced this module in the [terraform-modules-repository](https://github.com/jobteaser/terraform-modules)

### CODE REVIEW

**Review by zied-jt — COMMENTED** [review]
🍏

**Review by zied-jt — APPROVED** [review]
🍏

File: README.md [hunk 1]
```diff
@@ -69,7 +69,6 @@ $ make
 
 ## Maintainers
 
-This module is currently maintained by the individuals listed below.
+This module is a fork of [gearnode/terraform-kubernetes-get-secret](https://github.com/gearnode/terraform-kubernetes-get-secret).
 
-- [Bryan Frimin](https://github.com/gearnode)
-- [Ludovic Vielle](https://github.com/lukkor)
+We should open issues on the base repository if changes are needed.
```
Comment (jt-enrico) [inline README.md]: I am not so sure about this.

Comment (simonrdz) [reply README.md]: not sure either i'll remove it.

Comment (simonrdz) [inline README.md]:
```suggestion
```

**Pull request merged**